editions/tw5.com/tiddlers/filters/examples/addprefix.tid

```diff
@@ -1,7 +1,17 @@
 created: 20150118132851000
-modified: 20150118183116000
+modified: 20250323034523441
 tags: [[addprefix Operator]] [[Operator Examples]]
 title: addprefix Operator (Examples)
 type: text/vnd.tiddlywiki
 
 <<.operator-example 1 "Cat Garden [[Favourite Armchair]] +[addprefix[My ]]">>
+
```

Comment:
There are two issues here: the operator `(Examples)` tiddlers are intended for interactive examples. Instructional or reference documentation should go into the main `addprefix Operator` examples.

However, in this these instructions are not actually specific to the `addprefix` operator: they apply to all filter operators, and so this documentation belongs in one of the general filter documentation tiddlers. There is coverage of this topic in `Introduction to filter notation` and rather technical documentation in `Filter Parameter`. Neither of them are really adequate.

I suspect that we need a completely new tiddler on this topic. The current `Filters` tiddler is not particularly practical. Perhaps we can add some new, more user oriented documentation tiddlers linked from there.